src/components/shared/markdown/Markdown.jsx

```diff
@@ -3,31 +3,27 @@ import React from 'react';
 import ReactMarkdown from 'react-markdown';
 import Code from '../renderers/Code';
 import ListItem from '../renderers/ListItem';
-import {
-  injectCheckboxKeys,
-  updateCheckbox,
-} from '../../../helpers/checkbox-helpers';
+import {updateCheckbox} from './checkbox-helpers';
 
 type Props = {
   value: string,
   onChange: (value: string) => void,
 };
 
 const Markdown = (props: Props) => {
-  const onChangeCheckbox = (key, checked) => {
-    props.onChange(updateCheckbox(props.value, key, checked));
+  const onChangeCheckbox = (line, checked) => {
+    props.onChange(updateCheckbox(props.value, line, checked));
   };
 
   const renderListItem = props => (
     <ListItem {...props} onChangeCheckbox={onChangeCheckbox} />
   );
 
-  const checkboxifiedValue = injectCheckboxKeys(props.value);
-
   return (
     <ReactMarkdown
       className="markdown-body"
-      source={checkboxifiedValue}
+      source={props.value}
+      rawSourcePos={true}
```

Comment:
This seems to add a React warning about `sourcePos` being added to a tag element... It happens without any custom renderers too. I'll see if there is an issue logged in the repo.

Reply:
https://github.com/rexxars/react-markdown/pull/356